sound/smooth_swing_v2.h

```diff
@@ -95,6 +104,11 @@ class SmoothSwingV2 : public SaberBasePassThrough {
     // degrees per second
     // May not need to smooth gyro since volume is smoothed.
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
+    if (millis() - last_swing_millis_ >= 10 ) {
+      swing_acceleration_ = (speed - last_speed_) / (millis() - last_swing_millis_);
```

Comment:
We shouldn't be calculating the acceleration in two places.
Also, this calculation is significantly different from the one in the other code, which is not going to be good.

Reply:
Yes, I haven't updated the smooth_swing_v2.h file yet to match the work in hybrid_font.

Reply:
That seems like a reasonable way to do it.
Another way would be to move more of the smoothswing stuff into hybrid font and just send thresholds to it, not sure if that's any better though.

Reply:
all speed related code is migrated to hybrid_font.h StartSwing except for the speed stuff required for smooth swing v2.